ydb/docs/en/core/maintenance/manual/backup-collections.md

```diff
@@ -0,0 +1,311 @@
+
+## Creating backup collections {#creating-collections}
+
+### SQL Syntax Reference {#sql-syntax-reference}
+
+```sql
+-- CREATE BACKUP COLLECTION syntax
+CREATE BACKUP COLLECTION `collection_name`
+    ( TABLE `table_path` [, TABLE `table_path` ...] )
+WITH ( STORAGE = 'cluster', INCREMENTAL_BACKUP_ENABLED = 'true' );
+
+-- BACKUP syntax
+BACKUP `collection_name` [INCREMENTAL];
+
+-- DROP BACKUP COLLECTION syntax
+DROP BACKUP COLLECTION `collection_name`;
+
+-- RESTORE syntax
+
+RESTORE `collection_name`
+```
+
+For detailed syntax, see [YQL reference documentation](../../yql/reference/syntax/index.md).
+
+### Basic collection creation {#basic-collection-creation}
+
+Create a backup collection using SQL commands. The collection defines which tables to include and storage settings:
+
+```sql
+CREATE BACKUP COLLECTION `shop_backups`
+    ( TABLE `/Root/shop/orders`, TABLE `/Root/shop/products` )
+WITH ( STORAGE = 'cluster', INCREMENTAL_BACKUP_ENABLED = 'true' );
+```
+
+**Parameters:**
+
+- `STORAGE` - Storage backend for backups:
+  - `'cluster'` - Store backups within the YDB cluster (currently the only supported option)
+  - Future versions may support: `'object-storage'`, `'nfs'`, or other storage backends
+
+- `INCREMENTAL_BACKUP_ENABLED` - Enable incremental backup support:
+  - `'true'` - Enable incremental backups (recommended for production)
+  - `'false'` - Only full backups allowed
+
+## Taking backups {#taking-backups}
+
+### Initial full backup {#initial-full-backup}
+
+After creating a collection, take the initial full backup:
+
+```sql
+BACKUP `shop_backups`;
+```
+
+The first backup without the `INCREMENTAL` keyword creates a full backup containing all data from the specified tables.
+
+### Incremental backups {#incremental-backups}
+
+Once you have a full backup, create incremental backups to capture changes:
+
+```sql
+BACKUP `shop_backups` INCREMENTAL;
+```
+
+### Backup timing considerations {#backup-timing-considerations}
+
+Implement regular backup schedules based on your requirements. Note that scheduling must be implemented externally using cron or similar tools.
+
+**Daily full backups:**
+
+```sql
+-- Example: Run daily at 2 AM (requires external scheduling)
+BACKUP `shop_backups`;
+```
+
+**Hourly incrementals with weekly full backups:**
+
+```sql
+-- Example: Sunday: Full backup (requires external scheduling)
+BACKUP `shop_backups`;
+
+-- Example: Monday-Saturday: Incremental backups (requires external scheduling)
+BACKUP `shop_backups` INCREMENTAL;
+```
+
+### Important scheduling notes {#important-scheduling-notes}
+
+- **External scheduling required**: YDB does not provide built-in scheduling. Use cron or similar tools.
+- **Background operations**: Backups run asynchronously and don't block database operations.
```

Comment:
But they probably do consume some resources while they happen? How to avoid overruning the cluster with too many backups happening concurrently?